### ISSUE TITLE
False positive for use of new-ish aria-description attribute - supported in all major browsers

### ISSUE THREAD

**Issue by scottaohara** [IBMa/equal-access#2132]
One of the new features being added to ARIA 1.3 - the `aria-description` attribute allows authors to provide an element a description, similar to now `aria-label` allows authors to provide an element a name.

Added to the editor's draft in 2020, `aria-description` has been implemented in all browsers for quite awhile now.  it'd be nice if the equal access checker no longer reported its use as a violation due to being an invalid attribute.

thanks!

## Definition of Done

- [ ] Design artifacts reviewed
- [ ] Work item implemented
- [ ] Automated test case created (when applicable)
- [ ] Unit test completed
- [ ] Run Checker (when applicable)
- [ ] Manual and Screen Reader Testing (when applicable)
- [ ] Video recording of user experience walkthrough (if impact to UX)
- [ ] PR created

**Comment by tombrunet** [comment]
Just to be clear - this isn't part of the latest recommendation yet (ARIA 1.2). Due to end of year vacations, we probably won't be able to discuss this as a team for a couple of weeks.

**Comment by tombrunet** [comment]
Removing the label of false positive since it isn't a false positive by spec. More of a question of timing of draft spec features.

**Comment by shunguoy** [comment]
The checker misses some global ARIA attributes added in 1.3.

**Comment by tombrunet** [comment]
@scottaohara Do you have any other information about general support for aria-description?
https://a11ysupport.io/tech/aria/aria-description_attribute seems to imply that this is still not well supported.

**Comment by scottaohara** [comment]
@tombrunet it has the same support as `aria-describedby`

if you were to take the test from a11ysupport.io and change it to aria-describedby instead of aria-description, you'd get the same results.  (i say this because i just retested myself, which matches the behaviors from back in dec 2024)

you can look at the [aria-describedby tests](https://a11ysupport.io/tech/aria/aria-describedby_attribute) on there to see the similarities (even though aria-describedby was last updated 5 years ago).

The reason why aria-describedby might seem 'better supported' in looking at those test cases is that the aria-describedby attribute doesn't have the test of it being declared on a non-focusable span element, like the aria-description one does - and where all the "gaps" are being called out.  if you actually check the a11y tree in the browser, the description is exposed for both properties (because again, same thing...) but screen readers in scan/virtual cursor mode don't typically announce descriptions when navigating in that way.
